app/lib/services.rb

```diff
@@ -24,4 +24,11 @@ def self.publishing_api
       bearer_token: ENV.fetch("PUBLISHING_API_BEARER_TOKEN", "example"),
     )
   end
+
+  def self.search_api
+    @search_api = GdsApi::Search.new(
+      Plek.find("search"),
+      bearer_token: ENV.fetch("RUMMAGER_BEARER_TOKEN", "example"),
```

Comment:
You shouldn't need a bearer token here as this application will only be reading from Search API (like [finder-frontend](https://github.com/alphagov/finder-frontend/blob/main/app/lib/services.rb#L23-L25)).